src/node/routes/vscode.ts

```diff
@@ -24,7 +24,7 @@ export class CodeServerRouteWrapper {
     const isAuthenticated = await authenticated(req)
 
     if (!isAuthenticated) {
-      return redirect(req, res, "login", {
+      return redirect(req, res, "login/", {
```

Comment:
Note that the user won’t have to provide a trailing slash as we setup the redirect on their behalf.